code/__HELPERS/areas.dm

```diff
@@ -87,15 +87,20 @@ GLOBAL_LIST_INIT(typecache_powerfailure_safe_areas, typecacheof(/area/station/en
 	var/static/list/blacklisted_areas = typecacheof(list(
 		/area/space,
 		))
+
+	var/error = null
 	var/list/turfs = detect_room(get_turf(creator), area_or_turf_fail_types, BP_MAX_ROOM_SIZE*2)
-	if(!turfs)
-		to_chat(creator, span_warning("The new area must be completely airtight and not a part of a shuttle."))
-		return
-	if(length(turfs) > BP_MAX_ROOM_SIZE)
-		to_chat(creator, span_warning("The room you're in is too big. It is [length(turfs) >= BP_MAX_ROOM_SIZE *2 ? "more than 100" : ((length(turfs) / BP_MAX_ROOM_SIZE)-1)*100]% larger than allowed."))
+	var/turf_count = length(turfs)
+	if(!turf_count)
+		error = "The new area must be completely airtight and not a part of a shuttle."
+	else if(turf_count > BP_MAX_ROOM_SIZE)
+		error = "The room you're in is too big. It is [turf_count >= BP_MAX_ROOM_SIZE *2 ? "more than 100" : ((turf_count / BP_MAX_ROOM_SIZE)-1)*100]% larger than allowed."
+	if(!isnull(error))
```

Comment:
```suggestion
	if(error)
```

empty strings are falsey (in conjunction with the earlier review about not typing it)